README.md

```diff
@@ -24,6 +24,61 @@ cd $(The repo you just cloned)
 
 ## 2. Install all gems
 
+### Installation for Windows Users
+
+1. Install Ruby
+If you are on windows you will need to install `ruby`
+
+https://rubyinstaller.org/downloads/
+We suggest installing ruby version `2.6.6.1+dev kit`. Other versions
+may work but have not been tested.
+
+When you install ruby, be sure that you also check the box to install `MSYS2`.
+
+at the end of the `ruby` install, it will ask you if you want to install `MSYS2`.
+Install the base installation
+
+2. Next, install the bundler.
+
+`gem install bundler`
+
+
+Once you have completed this you can install all of the needed gems to run the
+website by cd'ing into the earthlab.github.io:
+
+`cd earthlab.github.io`
+
+then run:
+
+```
+# install the correct version of each gem specified in the gem file.
+# Run this inside the cloned earthlab.github.io directory
+bundle install
+```
+
+Once you have done this, and everything installs correctly, you are ready to
+build the website locally!
+
+** Note: you may need to run the above installations as an administrator on your
+machine.
```

Comment:
Should we also add notes telling them that they need to enable long paths on their machine, and that they might have to remove the images folder in order to run this code? Or are we thinking those will be fixed soon enough that we don't need instructions for them in this doc?

Reply:
oh to clone they will be the goal is to move away from needing to handle long paths... let's add this for now. @nkorinek do you want to directly edit this file in the pr to implement the changes above? If you can i'd really appreciate it!

Reply:
Sure I can add in the changes, if you're ok with all of them!

Reply:
your suggestions are all excellent @nkorinek ! please go ahead and edit the PR!! thank you

Reply:
@lwasser - the piece that I'm still confused about myself is when/how you actually build the site locally once I've made changes to the pages. so you edit a few pages, then do you open a new window in GitBash and run the line that creates it locally?

Reply:
oh @lherwehe let's make sure that is clear. i think ally asked a similar question. we can explain better in the readme. 

once you run `bundle exec jekyll serve` it creates almost a local server on your computer. so you can view the website by typing localhost:4000 into your web browser. and as you update pages, it updates the site! why don't we do a quick run through as a bigger group with ally one day!! in the meantime let's update this readme so it makes sense.

fresh eyes are exactly what this needs!